test/models/tipline_newsletter_test.rb

```diff
@@ -10,7 +10,7 @@ def setup
       rss_feed_url: 'https://example.com/feed',
       number_of_articles: 3,
       send_every: ['monday'],
-      send_on: Time.parse('2025-01-01'),
```

Comment:
LOL

Reply:
I spent some time thinking about the change in cached fields and why the newsletter tests fail until catch this line